cyctl/cmd/serve.go

```diff
@@ -0,0 +1,113 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"github.com/spf13/cobra"
+	"log"
+	"math/rand"
+	"net/http"
+	"net/url"
+	"path/filepath"
+	"strings"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
+	"k8s.io/client-go/util/homedir"
+)
+
+var serveCMD = &cobra.Command{
+	Use:   "serve",
+	Short: "Start the Cyclops UI",
+	Long:  "Start the Cyclops UI by forwarding the Cyclops UI service's port to a local port",
+	Run: func(cmd *cobra.Command, args []string) {
+		startPortForwarding()
+	},
+}
+
+func startPortForwarding() {
+	var kubeconfig *string
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.Parse()
+
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Can you instantiate clientset the same way it was done with [Moduleset](https://github.com/cyclops-ui/cyclops/blob/5fcddebac7e3d9fb93509b6844679c77aa914dfa/cyctl/internal/kubeconfig/kubeconfig.go#L13). You can then just pass it to `startPortForwarding` like it was done [here](https://github.com/cyclops-ui/cyclops/blob/f0d052ed8b735e0d8a418d88d86380342e7d3180/cyctl/internal/get/modules.go#L98)

Reply:
Like a global variable as the type `*kubernetes.Clientset` and then pass it to the function?

Reply:
Yeah, same thing as the Moduleset

Reply:
I was thinking of importing `kubeconfig` but it has Moduleset with a different type